third_party/gpus/nvidia_common_rules.bzl

```diff
@@ -668,3 +668,14 @@ def cuda_rpath_flags(relpath):
             ],
             "//conditions:default": [],
         })
+
+def if_cuda_major_version_newer_than(
+        major_version,
+        wanted_major_version,
+        if_true = [],
+        if_false = []):
```

Comment:
Thanks, I'll try this patch. One quick comment is that you probably don't want default values for if_true and if_false. The user should always specify. (Plus in this case the function needs to return a string but the defaults are lists.)